### PR TITLE
[HAL-1455] Add test coverage

### PR DESCRIPTION
**Pull request by mackaypeter**
Adding test coverage for https://issues.jboss.org/browse/HAL-1455

@zebrik could you please take a look? Thanks!

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/messaging/TopicMetricsTestCase.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    private static final Operations ops = new Operations(client);
+    private static final DeploymentsOperations deployOps = new DeploymentsOperations(client);
+
+    private static final String
+        SUBSYSTEM_NAME = "messaging-activemq",
+        SERVER = "server",
+        DEFAULT = "default",
+        TOPIC_NAME = "testTopic",
+        TOPIC_JNDI = "java:/jms/topic/" + TOPIC_NAME,
+        DEPLOYMENT_NAME = "producerServlet.war",
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/messaging/TopicMetricsTestCase.java]: What about to remove the IMHO redundant `.war` suffix of the `DEPLOYMENT_NAME`?

File: basic/src/test/resources/beans.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Marker file indicating CDI should be enabled -->
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="
+      http://xmlns.jcp.org/xml/ns/javaee
+      http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
+       bean-discovery-mode="all">
+</beans>
```
Comment (zebrik) [inline basic/src/test/resources/beans.xml]: You can add a newline at the end of the file.

**Review by zebrik — APPROVED** [review]
Looks good. Thank you! Just cosmetic optional suggestions.

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/messaging/TopicMetricsTestCase.java (hunk 1 above)
Comment (mackaypeter) [reply basic/src/test/java/org/jboss/hal/testsuite/test/runtime/messaging/TopicMetricsTestCase.java]: Yeah I had to split the name into name + extension and forgot to remove the extension, thanks for spotting it.

**Pull request merged**